contracts/stabilizer/PSMRouter.sol

```diff
@@ -32,15 +69,61 @@ contract PSMRouter is IPSMRouter {
     /// @dev This wraps ETH and then calls into the PSM to mint the fei. We return the amount of fei minted.
     /// @param to The address to mint fei to
     /// @param minAmountOut The minimum amount of fei to mint
-    /// @param deadline The minimum amount of fei to mint
+    /// @param deadline The deadline for this order to be filled
     function mint(address to, uint256 minAmountOut, uint256 deadline) external payable ensure(deadline) returns (uint256) {
```

Comment:
I personally like forcing the caller to specify the ETH amount to be explicit and require they are equal, but this is a style choice and costs more gas

Reply:
Added.